test/protocol-buffer-conversion-test.ts

```diff
@@ -166,4 +182,49 @@ describe('Protocol Buffer Conversion', function(): void {
     // THEN the result is undefined
     assert.isUndefined(issuedCurrency)
   })
+
+  // CurrencyAmount tests
+  it('Convert CurrencyAmount with drops', function(): void {
+    // GIVEN a currency amount protocol buffer with an XRP amount.
+    const drops = '10'
+    const currencyAmountProto = new CurrencyAmount()
+    const xrpDropsAmountProto = new XRPDropsAmount()
+    xrpDropsAmountProto.setDrops(drops)
+    currencyAmountProto.setXrpAmount(xrpDropsAmountProto)
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const currencyAmount = XRPCurrencyAmount.from(currencyAmountProto)
+
+    // THEN the result has drops set and no issued amount.
+    assert.isUndefined(currencyAmount?.issuedCurrency)
+    assert.equal(currencyAmount?.drops, drops)
+  })
+
+  it('Convert CurrencyAmount with Issued Currency', function(): void {
+    // GIVEN a currency amount protocol buffer with an issued currency amount.
+    const currencyAmountProto = new CurrencyAmount()
+    currencyAmountProto.setIssuedCurrencyAmount(testIssuedCurrency)
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const currencyAmount = XRPCurrencyAmount.from(currencyAmountProto)
+
+    // THEN the result has an issued currency set and no amount.
+    assert.deepEqual(
+      currencyAmount?.issuedCurrency,
+      XRPIssuedCurrency.from(testIssuedCurrency),
+    )
+    assert.isUndefined(currencyAmount?.drops)
+  })
+
```

Comment:
Can you add a test case for converting with the bad issued amount proto you made on line 42? Analagous to https://github.com/xpring-eng/XpringKit/pull/130/files#diff-9307f8a129e7063d9086218b91281a2bR154

Reply:
I think this test is already in here, but I just refactored it to use the testInvalidIssuedCurrency instead of re-creating it.